.github/action_scripts/provideNewNightlyBundle.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set -ex
+
+ROOT_PROJECT_DIR="${GITHUB_WORKSPACE}"
+if [ -z "${ROOT_PROJECT_DIR}" ]; then
+  BASE_DIR=$(cd $(dirname "$0"); pwd)
+  ROOT_PROJECT_DIR=$(dirname "$(dirname "${BASE_DIR}")")
+fi
+
+installOperatorSDK() {
+  YQ=$(command -v operator-sdk) || true
```

Comment:
It would be nice to rename this variable to the OPERATOR_SDK... YQ it's a cli tool to parse yaml files.